aws/src/test/java/org/apache/iceberg/aws/s3/TestS3FileIOProperties.java

```diff
@@ -254,6 +255,11 @@ public void testS3FileIOProperties() {
         .containsEntry(
             S3FileIOProperties.REMOTE_SIGNING_ENABLED,
             String.valueOf(s3FileIOProperties.isRemoteSigningEnabled()));
+
+    Assertions.assertThat(map)
+        .containsEntry(
+            S3FileIOProperties.WRITE_STORAGE_CLASS,
+            StorageClass.INTELLIGENT_TIERING.toString());
```

Comment:
it would be better to just pass a plain string rather than adding a dependency to `com.adobe.testing.s3mock.dto.StorageClass`